src/java.base/share/classes/sun/security/ssl/ClientHello.java

```diff
@@ -430,8 +430,8 @@ public byte[] produce(ConnectionContext context) throws IOException {
 
                 if (!session.isRejoinable()) {
                     session = null;
-                    if (SSLLogger.isOn &&
-                            SSLLogger.isOn("ssl,handshake,verbose")) {
+                    if (SSLLogger.logging &&
+                            SSLLogger.isOn(SSLLogger.Opt.HANDSHAKE_VERBOSE)) {
```

Comment:
Not your fault, but I'm not sure I like this as a verbose message.

Reply:
In agreement. This sort of message strikes me as more in the `HANDSHAKE `category

Will I log a follow on JBS issue to have all call sites of `Opt `examined ?